SolrNet/Commands/LukeCommand.cs

```diff
@@ -0,0 +1,21 @@
+using SolrNet;
```

Comment:
Classes implementing `ISolrCommand` now also need to implement `Task<string> ExecuteAsync(ISolrConnection connection);`